### ISSUE TITLE
Drawing canvas doesn't scale with window

### ISSUE THREAD

**Issue by sparticvs** [OWASP/threat-dragon#615]
**Describe the bug**
When the window is resized, the DFD drawing canvas doesn't resize with it.

**Expected behaviour**
Expected the canvas to size to fill the window (or to have the ability to resize the panes in the window.

**Environment**
 - Version: 2.0
 - Platform: AppImage
 - OS: RHEL 8.6
 - Browser: all

**To Reproduce**
1. Open a threat model
2. Click into the diagram
3. Resize the window

**Any additional context, screenshots, etc**
![Screenshot from 2023-03-06 10-33-54](https://user-images.githubusercontent.com/847020/223156463-32b283a6-baf6-4b83-b5e8-7f3902e69075.png)

**Comment by ChristopherHackett** [comment]
Can also be reproduced on threatdragon.com (v2.0.1-demo). Suspect issue affects all environments.

**Comment by jgadsden** [comment]
thanks @sparticvs , this is a very good point
I have labeled it as something for version 2.0, in the hope that we can sort it for then

**Comment by jgadsden** [comment]
not able to work out how to implement this: https://x6.antv.vision/en/docs/api/graph/graph#autoresize

The work around is to close the diagram and open it again - it then resizes for the new window size

**Comment by jgadsden** [comment]
Need to document the workaround
attempt to correct this did not work: see 5b5b48b62ccb450dd8fb5d16f94e9c65a8f43c3d

**Comment by Narasimha9271** [comment]
Can I work on this issue

**Comment by raghav1030** [comment]
hey @jgadsden! I have tried to implement auto resizing as described in https://x6.antv.vision/en/docs/api/graph/graph#autoresize but haven't been able to. I am still trying to get a way out. Meanwhile I have implemented the window resizing issue that is the core problem as described in this raised issue.

here's what the update does : 


https://github.com/OWASP/threat-dragon/assets/94848646/9fa7f9c1-c500-455b-b9c7-c5c33b5453be
